File: LAB 6/C++/lab6_1.cpp
#include <iostream>
#include <cmath>

using namespace std;
double d = 0;

int main() {

    int donut;

    cout << "Enter a number for n: ";
    cin >> donut;

    sum_recursive(donut);
    cout << "......Sum of the given equation is: " << d << endl;
    cout << "...Ty for choosing us...";
    return 0;
}

void sum_recursive(int donut) {

    if (donut == 1) {
        d += 1;
        return;

    } else {
        sum_recursive(donut - 1);
        d += pow(-1, donut + 1) / donut;
        return;
    }
}

File: LAB 6/Python/lab6_1.py
from math import factorial

n = int(input("Enter a number For n: "))
x = int(input("Enter a number For x: "))

donut = lambda n, x: sum([n ** d / factorial(d) for d in range(x + 1)])

answer = 1 + donut(n, x)
print(f"e^{n} = {answer}")
print("...Ty For choosing Us...")
